docs/runKafka.txt

```diff
@@ -0,0 +1,28 @@
+* Get kafka
+[[https://downloads.apache.org/kafka/3.2.1/kafka_2.13-3.2.1.tgz][link]]
+* unpack
+* adjust kafka_2.13-3.2.1/config/zookeeper.properties
+change dataDir to existing directory (create it if it does not exist)
+for example:
+dataDir=c:/Users/vanisp/tmp/zookeeper 
+* adjust kafka_2.13-3.2.1/config/server.properties
+change log.dirs to existing directory (create it if it does not exist)
+for example:
+log.dirs=c:/Users/vanisp/tmp/kafka-logs
+uncomment and change listeners
+listeners=PLAINTEXT://127.0.0.1:9092
+* following commands to be run from kafka home
+example:  C:\Users\vanisp\bin\kafka_2.13-3.2.1\
```

Comment:
It cause an error if the path of kafka is too long. ( 25 chars leng -> error at building of kafka class path )